tests/test_utils.py

```diff
@@ -79,3 +79,14 @@ def test_ingest_parallaxes(db, t):
     assert results['source'][0] == 'Fake 3'
     assert results['parallax'][0] == 155
     assert results['parallax_error'][0] == 0.6
+
+def test_add_publication(db):
```

Comment:
This is breaking the tests. Either complete the test, or add an explicit `return` to continue